app/src/main/java/umc/onairmate/ui/friend/search/SearchFriendTabFragment.kt

```
@@ -22,9 +34,59 @@ class SearchFriendTabFragment: Fragment() {
+        adapter = SearchUserRVAdapter() { data ->
+            val text = data.nickname+"님에게 친구요청을 보내시겠습니까?"
+            val textList = listOf(text,"예","아니오")
+            val dialog = TwoButtonPopup(textList,object : PopupClick{
+                override fun rightClickFunction() {
+                    // 실행하고자 하는 함수 있으면 overriding
+
+                }
+
+                override fun leftClickFunction() {
+
+                }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)